challenge-06/challenge-06.js

```diff
@@ -32,19 +32,36 @@ Crie uma função chamada `showTeamPosition` com as seguintes características:
     - Se não houver time para a posição passada, deve retornar a mensagem:
     "Não temos a informação do time que está nessa posição."
 */
-// ?
+function showTeamPosition(num){
+    
+    if (num > 5 || num === undefined || num === 0) {
+        return 'Não temos a informação do time que está nessa posição.';
+    };
 
+    posicao = num;
+    time    = num - 1;
+
+    return 'O time que está em ' + posicao + 'º lugar é o ' + teams[time]+ '.';
+}
 /*
 Escolha 4 times do campeonato selecionado e mostre a posição dele, usando a
 função acima. Entre esses 4, adicione 1 que não esteja entre os 5 primeiros.
 */
-// ?
+console.log( showTeamPosition(1) );
+console.log( showTeamPosition(3) );
+console.log( showTeamPosition(6) );
+console.log( showTeamPosition(2) );
 
 /*
 Mostre os números de 20 a 30 no console (inclusive o 30), usando a estrutura de
 repetição "while".
 */
-// ?
+var count = 20;
+
+while(count > 20 && count < 30){
```

Comment:
-1
@Feabbud não tem como o contador ser menor que 20, pois já está atribuindo esse valor a ele, e dentro do loop, você está incrementando. Então pode remover a verificação se o contador é maior que 20. Só teste enquanto ele for menor que 30 :)